### PR TITLE
tests: update test wagmi contract

### PR DESCRIPTION
**Pull request by jxom**
## Description

This PR updates the `wagmiContractConfig` to use the new wagmi mint example contract. Since the new contract supports a `tokenId` argument to the `mint` function, we can remove the `getTotalSupply` test util.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -162,7 +162,7 @@ jobs:
         run: anvil --fork-url $ANVIL_FORK_URL --fork-block-number $ANVIL_BLOCK_NUMBER &
         env:
           ANVIL_FORK_URL: ${{ secrets.ANVIL_FORK_URL }}
-          ANVIL_BLOCK_NUMBER: 15132000
+          ANVIL_BLOCK_NUMBER: 15564160
```
Comment (jxom) [inline .github/workflows/main.yml:165]: Updating the block number so we can discover the contract.

File: packages/core/src/actions/accounts/fetchBalance.test.ts [hunk 2]
```diff
@@ -19,10 +19,10 @@ describe('fetchBalance', () => {
         ).toMatchInlineSnapshot(`
           {
             "decimals": 18,
-            "formatted": "1.889009973656812885",
+            "formatted": "0.40742480512617271",
```
Comment (jxom) [inline packages/core/src/actions/accounts/fetchBalance.test.ts:22]: Side effect from updating block number.

File: packages/react/src/hooks/accounts/useBalance.test.ts [hunk 3]
```diff
@@ -19,10 +19,10 @@ describe('useBalance', () => {
       {
         "data": {
           "decimals": 18,
-          "formatted": "1.889009973656812885",
+          "formatted": "0.40742480512617271",
```
Comment (jxom) [inline packages/react/src/hooks/accounts/useBalance.test.ts:22]: Side effect from updating block number.

File: packages/core/test/utils.ts [hunk 4]
```diff
@@ -233,3 +215,7 @@ export function getCrowdfundArgs({
   }
   return Object.values(data)
 }
+
+export function getRandomTokenId() {
+  return BigNumber.from(Math.floor(Math.random() * 1000) + 69420)
```
Comment (tmm) [inline packages/core/test/utils.ts:220]: don't want to use `bigint` 😛

**Review by tmm — APPROVED** [review]

**Pull request merged**